### PR TITLE
Rename method arguments to use camelCase casing (module directories)

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 902fa431bac2c197b66e44484f1a0e3ed1de72aa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31150">issues.joomla.org/tracker/joomla-cms/31150</a>.</sub>

File: modules/mod_search/helper.php [hunk 1]
```diff
@@ -19,14 +19,14 @@ class ModSearchHelper
 	/**
 	 * Display the search button as an image.
 	 *
-	 * @param   string  $button_text  The alt text for the button.
+	 * @param   string  $buttonText  The alt text for the button.
```
Comment (Quy) [inline modules/mod_search/helper.php]: `$buttonAltText`?

Comment (zero-24) [reply modules/mod_search/helper.php]:
```suggestion
	 * @param   string  $buttonAltText  The alt text for the button.
```

File: modules/mod_search/helper.php [hunk 2]
```diff
@@ -19,14 +19,14 @@ class ModSearchHelper
...
 	 *
 	 * @return  string  The HTML for the image.
 	 *
 	 * @since   1.5
 	 */
-	public static function getSearchImage($button_text)
+	public static function getSearchImage($buttonText)
```
Comment (zero-24) [inline modules/mod_search/helper.php]:
```suggestion
	public static function getSearchImage($buttonAltText)
```

File: modules/mod_search/helper.php [hunk 3]
```diff
@@ -19,14 +19,14 @@ class ModSearchHelper
...
 	{
-		return JHtml::_('image', 'searchButton.gif', $button_text, null, true, true);
+		return JHtml::_('image', 'searchButton.gif', $buttonText, null, true, true);
```
Comment (zero-24) [inline modules/mod_search/helper.php]:
```suggestion
		return JHtml::_('image', 'searchButton.gif', $buttonAltText, null, true, true);
```

File: modules/mod_search/helper.php (hunk 1 above)
Comment (SharkyKZ) [reply modules/mod_search/helper.php]: This actually is not used and can be removed.

**Comment by zero-24** [comment]
@SharkyKZ i'm not sure whether we can remove that argument from a public method?

**Comment by SharkyKZ** [comment]
Calling a method with more arguments than it accepts does not cause issues.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 306cfb9adb58f0192df43b536e4ccc1fcdc7e9e6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31150">issues.joomla.org/tracker/joomla-cms/31150</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 306cfb9adb58f0192df43b536e4ccc1fcdc7e9e6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31150">issues.joomla.org/tracker/joomla-cms/31150</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31150">issues.joomla.org/tracker/joomla-cms/31150</a>.</sub>

**Pull request merged**